tardis/montecarlo/tests/test_montecarlo_line_scatter.py

```diff
@@ -0,0 +1,20 @@
+import os
+from ctypes import CDLL, c_double, c_longdouble
+
+import pytest
+
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+
+tests = CDLL(test_path)
+
+def test_compute_distance2boundary():
+	tests.test_compute_distance2boundary.restype = c_longdouble
```

Comment:
Ohh yes, sorry I updated in my local, just forgot to push those changes. Trying out some stuff, will push in a while @wkerzendorf .